deploy/helm/templates/deployment.yaml

```diff
@@ -0,0 +1,89 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "grafana-operator.fullname" . }}
+  labels: {{- include "grafana-operator.labels" . | nindent 4 }}
+spec:
+  replicas: 1
+  selector: 
+    matchLabels: {{- include "grafana-operator.matchLabels" . | nindent 6 }}
+  strategy:
+    type: Recreate
```

Comment:
@Cellebyte can we have ```rollingUpdate``` strategy type ? Any specific reason of using Recreate :)